lib/realtime_web/live/inspector_live/conn_component.html.heex

```diff
@@ -104,7 +104,21 @@
                     focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50
                   " %>
             <%= error_tag f, :token %>
-            <p class="text-gray-600 text-xs italic">Your Supabase `anon_key` or Auth user JWT</p>
+            <p class="text-gray-600 text-xs italic">Your Supabase `anon` or `service_role` key</p>
+        </div>
+        <div class="mb-4">
+            <%= label f, :bearer, class: "block text-gray-700 text-sm font-bold mb-2" %>
+            <%= text_input f, :bearer, value: (Ecto.Changeset.get_field(@changeset, :bearer)), class: "
+                    my-1
+                    block
+                    w-full
+                    rounded-md
+                    border-gray-300
+                    shadow-sm
+                    focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50
+                  " %>
+            <%= error_tag f, :bearer %>
+            <p class="text-gray-600 text-xs italic">An Auth user JWT optionally signed with custom claims</p>
```

Comment:
@chasers if the only purpose is to pass in a JWT with custom claims then I would just be explicit and mention that

```suggestion
            <p class="text-gray-600 text-xs italic">A manually signed JWT with your custom claims</p>
```

or something to that effect.